src/track/libheaptrack.cpp

```diff
@@ -261,15 +261,6 @@ class HeapTrack
         static once_flag once;
         call_once(once, [] {
             debugLog<MinimalOutput>("%s", "doing once-only initialization");
-            // configure libunwind for better speed
```

Comment:
if libunwind isn't available on android, I'm OK with making the backtrace backend optional. abstract setup code and introduce separate files, e.g. `backtrace_libunwind.cpp` and `backtrace_unwind_tables.cpp`, then compile in either of them based on a CMake option. 

Here, we'd then call something like

backtrace::setup();

which would do nothing in the unwind tables case, and would call the code you removed for the libunwind scenario